### PR TITLE
[ERSSUP-68528]-[]-[APIM OAS bug fixes - Sprint 210]-[DMW]

### PR DESCRIPTION
**Pull request by nhsd-david-wass**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/components/stu3/schemas/endpoints/a020-upload-file-to-document-store.yaml [hunk 1]
```diff
@@ -56,7 +56,6 @@ description: |
 summary: A020 - Upload file to document store
 operationId: a020-upload-file-to-document-store
 parameters:
-  - $ref: '../headers/request/ContentTypeAttachmentsUpload.yaml'
```
Comment (nhsd-jack-wainwright) [inline specification/components/stu3/schemas/endpoints/a020-upload-file-to-document-store.yaml:59]: Now that this parameter has been removed is the `ContentTypeAttachmentsUpload` component still used elsewhere? With the removal of this parameter is the need for a `content-type` header still documented?

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Looks good to me 🙂, just a few minor comments.

**Review by stuandgray — APPROVED** [review]

**Pull request merged**